org.eclipse.wildwebdeveloper/pom.xml

```diff
@@ -55,6 +66,7 @@
 							<executable>npm</executable>
 							<arguments>
 								<arg>install</arg>
+								<arg>--no-bin-links</arg>
```

Comment:
That's interesting. Maybe there is an option to copy the binaries instead of linking them?